src/Controls/src/Core/SwipeView/SwipeView.cs

```diff
@@ -346,8 +361,8 @@ void UpdateMargin()
 
 		void OnParentScrolled(object? sender, ScrolledEventArgs e)
 		{
-			var horizontalDelta = e.ScrollX - _previousScrollX;
-			var verticalDelta = e.ScrollY - _previousScrollY;
+			var horizontalDelta = Math.Abs(e.ScrollX - _previousScrollX);
```

Comment:
Did you test this change with SwipeItems in each direction?

Reply:
Yes, I did. That's how I discovered that without Math.Abs the scroll view behaves strangely. I tested on this project: https://github.com/user-attachments/files/16887019/MauiApp12.zip

Reply:
Could you include an UITest based on that sample? let me know if need help or if want me to include the test etc.